### DISCUSSION TITLE
How to use the model with your own data?

### DISCUSSION THREAD

**Discussion by JuliaWasala** [fuzailpalnak/building-footprint-segmentation#27] [Q&A]
Hi there, I was wondering how I can use my own data. I haven't been able to find out yet from the source code how to do it!

**Comment by fuzailpalnak** [comment]
Check out the [example](https://github.com/fuzailpalnak/building-footprint-segmentation/tree/main/examples)  notebook which contains on how to train a network

**Reply by fuzailpalnak** [reply]
And of you want to use the model present in the repo for inference, check out the [issue](https://github.com/fuzailpalnak/building-footprint-segmentation/issues/23#issuecomment-759915555)
